View/View.php

```diff
@@ -25,6 +25,7 @@
 class View
 {
     private $data;
+    private $templateData = array();
```

Comment:
Minor fix: it's better to move this initialization to the constructor, since it is already present